src/Extension.ml

```diff
@@ -61,8 +61,8 @@ let selectSandbox (instance : Instance.t) () =
 let suggestToSetupToolchain instance =
   let open Promise.O in
   Vscode.Window.showInformationMessage'
-    "There is no toolchain attached to this project."
-    [ ("select toolchain", ()) ]
+    "Extension is unable to find ocamllsp automatically. Please select package manager you used to install ocamllsp for this project."
```

Comment:
Could you format the code with ocamlformat?

Reply:
Formatting triggers too many unrelated changes, I did in separate PR: https://github.com/ocamllabs/vscode-ocaml-platform/pull/152

Reply:
Formatted added string: https://github.com/ocamllabs/vscode-ocaml-platform/pull/149/commits/e8f5755c55c520725b135f4064840a339c31919a